File: framework/core/platformwrapper/android/billing/BillingObserver.java
package org.akkord.lib;

public interface BillingObserver {
    public void BillingSetupFinished(int ResponseCode);
    public void BillingDisconnected();
    public void PurchaseQueried(String PurchaseToken, String ProductSKU, int Type);
    public void PurchaseConsumed(String PurchaseToken, String ProductSKU);
}

File: framework/core/platformwrapper/android/billing/BillingObserver.kt
package org.akkord.lib

interface BillingObserver {
    fun onBillingSetupFinished()
    fun onBillingDisconnected()
    fun onPurchaseQueried(purchaseToken: String?, productSKU: String?, type: Int)
    fun onPurchaseConsumed(purchaseToken: String?, productSKU: String?)
}